server/db.js

```diff
@@ -14,85 +14,94 @@ const config = {
   dropDatabase: true,
 };
 
-// schema of each individual review
-const reviewSchema = new mongoose.Schema({
-  id: mongoose.ObjectId,
-  user_icon: String,
-  review_text: String,
-  date: Date,
-  name: String,
-  rating: {
-    cleanliness: Number,
-    communication: Number,
-    check_in: Number,
-    accuracy: Number,
-    location: Number,
-    value: Number
-  }
-});
-
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function() {
-  // populate the database once we are connected
-  initDB();
```

Comment:
Dont call initDB() in the open, as it will be called more than once and create too much data.

Reply:
https://github.com/keybox130/Reviews-Service/pull/1/files/0b9b2ebdee1f09bca326960678f7fc69a9b5ba3c..014a120f3f4739003f7d084f60d9fe05b115aaec#r495200408